package.json

```diff
@@ -21,44 +21,40 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.1.2",
-    "ember-modifier-manager-polyfill": "^1.0.3"
+    "ember-cli-babel": "^7.11.1",
+    "ember-modifier-manager-polyfill": "^1.1.0"
   },
   "devDependencies": {
-    "@ember/optional-features": "^0.6.3",
-    "broccoli-asset-rev": "^2.7.0",
-    "ember-cli": "~3.8.1",
+    "@ember/optional-features": "^1.0.0",
+    "babel-eslint": "^10.0.3",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-cli": "~3.14.0",
     "ember-cli-content-security-policy": "^1.1.1",
     "ember-cli-dependency-checker": "^3.1.0",
-    "ember-cli-eslint": "^4.2.3",
-    "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
-    "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-eslint": "^5.1.0",
+    "ember-cli-htmlbars": "^4.0.8",
+    "ember-cli-inject-live-reload": "^2.0.1",
     "ember-cli-release": "^1.0.0-beta.2",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-template-lint": "^1.0.0-beta.1",
-    "ember-cli-uglify": "^2.1.0",
+    "ember-cli-template-lint": "^1.0.0-beta.3",
+    "ember-cli-uglify": "^3.0.0",
     "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.0",
```

Comment:
Why did you removed `ember-export-application-global`. It's still part of default blueprint, isn't it?

Reply:
Oh, I thought that it is not used. If you think it is useful, I can bring it back.

Reply:
To be honest, I don't know if it's needed but would feel more comfortable if we stick to default blueprints as much as possible to avoid unnecessary complications.

Reply:
This addon is supposed to be used for debugging. I think it is not needed in this addon, it is safe to delete it. But I can bring it back tomorrow if you still think it is needed.

Reply:
Brought it back